internal/azure-identity/deployment/src/main/java/io/quarkiverse/azure/identity/deployment/AzureIdentityProcessor.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkiverse.azure.identity.deployment;
+
+import java.util.stream.Stream;
+
+import io.quarkus.deployment.annotations.BuildProducer;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.deployment.builditem.nativeimage.NativeImageResourceBuildItem;
+import io.quarkus.deployment.builditem.nativeimage.RuntimeInitializedClassBuildItem;
+import io.quarkus.deployment.pkg.builditem.NativeImageRunnerBuildItem;
+import io.quarkus.deployment.pkg.steps.NativeOrNativeSourcesBuild;
+
+public class AzureIdentityProcessor {
+    @BuildStep(onlyIf = NativeOrNativeSourcesBuild.class)
+    void runtimeInitializedClasses(BuildProducer<RuntimeInitializedClassBuildItem> runtimeInitializedClass) {
+        Stream.of(
+                "com.microsoft.aad.msal4jextensions.persistence.linux.ISecurityLibrary",
+                "com.microsoft.aad.msal4jextensions.persistence.mac.ISecurityLibrary")
+                .map(RuntimeInitializedClassBuildItem::new)
+                .forEach(runtimeInitializedClass::produce);
+    }
+
+    @BuildStep(onlyIf = NativeOrNativeSourcesBuild.class)
+    public void resources(
+            BuildProducer<NativeImageResourceBuildItem> resource,
+            NativeImageRunnerBuildItem nativeImageRunnerFactory) {
+
+        String dir = "QMETA-INF/services";
+        String hotspotLibName = "jdk.vm.ci.hotspot.HotSpotJVMCIBackendFactory";
```

Comment:
Add a comment explaining what the `QMETA-INF/services` thing is.

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/218/commits/37c8dc9fcc6876ff74ea2951c475e93dd84765ad.